### PR TITLE
added support to get capabilities for clusterversion

### PR DESCRIPTION
**Pull request by pruan-rht**
helper for #3623
example run.
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/Runner-v3-smoke/7581/console

### CODE REVIEW

**Comment by pruan-rht** [comment]
@liangxia @xingxingxia PTAL.   @dpuniaredhat, you can look at the test feature to see an example on how to call the method.

File: features/test/new_api.feature [hunk 1]
```diff
@@ -27,3 +27,8 @@ Feature: test new api methods
     Given I switch to cluster admin pseudo user
     And I use the "openshift-monitoring" project
     And evaluation of `route('prometheus-k8s').spec.host` is stored in the clipboard
+
+  @admin
+  Scenario: check cluster_version capabilities
+    And the expression should be true> cluster_version('version').capability_is_enabled?(capability: 'ImageRegistry')
```
Comment (dpuniaredhat) [inline features/test/new_api.feature:33]: @pruan-rht Is skip this test if `ImageRegistry` capabilities are not enabled to avoid failure.

Comment (pruan-rht) [reply features/test/new_api.feature:33]: @dpuniaredhat this is just an example of how to use it.  Once this PR is merged, then you can update your PR to use `cluster_version('version').capability_is_enabled?(capability: 'ImageRegistry')` in your step_definition 
```
unless is_capability_enabled?(capability)
```

Comment (dpuniaredhat) [reply features/test/new_api.feature:33]: Thanks, Got it

File: features/step_definitions/cluster_version.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+Given(/^the "([^"]*)" capability is enabled$/) do |capability|
```
Comment (pruan-rht) [inline features/step_definitions/cluster_version.rb:2]: @dpuniaredhat I've added the step definition here.  You can now remove the step definition in your PR and just use this one instead.

Comment (dpuniaredhat) [reply features/step_definitions/cluster_version.rb:2]: Thanks

**Pull request merged**